test/Concurrency/grouped_actor_isolation_diagnostics.swift

```diff
@@ -81,3 +111,15 @@ struct HasWrapperOnActor {
     testMA() // expected-error{{call to main actor-isolated instance method 'testMA()' in a synchronous nonisolated context}}
   }
 }
+
+@preconcurrency @MainActor
+class MainActorPreconcurrency {}
+
+class InferMainActorPreconcurrency: MainActorPreconcurrency {
+  static func predatesConcurrency() {}
+}
+
+nonisolated func testPreconcurrency() {
+  InferMainActorPreconcurrency.predatesConcurrency()
+  // expected-warning@-1 {{call to main actor-isolated static method 'predatesConcurrency()' in a synchronous nonisolated context}}
```

Comment:
Can you please add a test that has multiple callsites to `@preconcurrency` functions to ensure the diagnostic is still downgraded in that case?